pkg/koordlet/resmanager/cgroup_reconcile.go

```diff
@@ -316,13 +319,6 @@ func (m *CgroupResourcesReconcile) calculateContainerResources(container *corev1
 			klog.V(5).Infof("correct calculated memory.low for container since it is lower than memory.min,"+
 				" pod %s, container %s, current value %v", util.GetPodKey(pod), container.Name, *summary.memoryLow)
 		}
-		// values improved: memory.high is no less than memory.min
```

Comment:
Do we need to keep this fixup?

Reply:
1. `memory.min` := spec.requests.memory * minLimitFactor / 100 (use 0 if requests.memory is not set)
4. `memory.high` := floor[(spec.requests.memory + throttlingFactor / 100 * (memory.limit_in_bytes or node allocatable memory - spec.requests.memory))/pageSize] * pageSize

Theoretically, this branch will not be reached unless spec.requests.memory < pagesize(default 4k)

In order to avoid extreme situations, I still fall back

Reply:
OK. Please note that the `minLimitFactor` and `throttlingFactor` are variables.